thoth/worker/tasks/github.py

```diff
@@ -21,51 +21,49 @@
 import logging
 from urllib.parse import urlparse
 from collections import OrderedDict
+from requests.api import head
 
 from selinon import SelinonTask
 from selinon import StoragePool
 from selinon import FatalTaskError
 
 import requests
 from thoth.python import Source
+import yaml
 
 _LOGGER = logging.getLogger(__name__)
 _LOGGER.setLevel(logging.DEBUG)
 
 
 PYPI = Source(url="https://pypi.org/simple")
+GITHUB_TOKEN = os.getenv("GITHUB_TOKEN")
+_PRESCRIPTIONS_BASE_URL = "https://raw.githubusercontent.com/thoth-station/prescriptions/master/prescriptions/"
 
 
 class _GitHubTaskBase(SelinonTask):
     """A base class for GitHub related routines."""
 
     def get_project_repo_github(self, package_name: str):
-        """Get project and repo for the given package based on aggregated package info from PyPI."""
-        project_info_store = StoragePool.get_connected_storage("ProjectInfoStore")
-
-        home_page = (
-            project_info_store.retrieve_project_info(package_name)
-            .get("info", {})
-            .get("home_page")
+        """Get project and repo for the given package based on aggregated package info from Thoth's Prescriptions."""
+        package_prefix = package_name[:2] + "_"
+        headers = {"Authorization": GITHUB_TOKEN}
+
+        gh_link_yaml = requests.get(
+            os.path.join(
+                _PRESCRIPTIONS_BASE_URL, package_prefix, package_name, "gh_link.yaml"
+            ),
+            headers=headers,
         )
-        if not home_page:
-            raise FatalTaskError(f"No home page found for project {package_name!r}")
 
-        home_page = urlparse(home_page)
-        if home_page.netloc != "github.com":
-            raise FatalTaskError(
-                f"No GitHub home page associated for project {package_name!r}"
+        if gh_link_yaml.status_code != 200:
+            _LOGGER.debug(
+                f"{gh_link_yaml.status_code}: Cannot access prescription YAML file"
```

Comment:
If `status_code` != 200 then this will continue and `yaml.safe_load` will not load prescription